openquakeplatform/openquakeplatform/test/platform.py

```diff
@@ -0,0 +1,233 @@
+import time
+from utils import *
+
+
+class Platform(object):
+    DT = 0.1
+
+    def __init__(self):
+
+        self.driver = None
+        self.basepath = None
+        self.user = None
+        self.passwd = None
+        self.debugger = None
+
+    def init(self):
+        try:
+            from openquakeplatform.test.config import (
+                pla_basepath, pla_user, pla_passwd, pla_debugger)
```

Comment:
I guess there is a reason why this import is inside the init and not at top level. Can we explain it in a comment?